Error during dotnet clean trying to use task that has been clean

```
jose@mac icerpc-csharp % dotnet clean
  IceRpc.Protobuf.Tests net10.0 failed with 1 error(s) (0,1s)
    /Users/jose/Documents/IceRPC/icerpc-csharp/src/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.targets(107,5): error MSB4062: The "IceRpc.Protobuf.Tools.OutputFileNamesTask" task could not be loaded from the assembly /Users/jose/Documents/IceRPC/icerpc-csharp/src/IceRpc.Protobuf.Tools/bin/Debug/net10.0/IceRpc.Protobuf.Tools.dll. Could not find file '/Users/jose/Documents/IceRPC/icerpc-csharp/src/IceRpc.Protobuf.Tools/bin/Debug/net10.0/IceRpc.Protobuf.Tools.dll'. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.

Build failed with 1 error(s) in 0,6s
```

--- a/src/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.targets
+++ b/src/IceRpc.Protobuf.Tools/IceRpc.Protobuf.Tools.targets
@@ -103,7 +103,7 @@
       />
     </ItemGroup>
   </Target>
-  <Target Name="ProtoClean" BeforeTargets="Clean">
+  <Target Name="ProtoClean" BeforeTargets="Clean" Condition="Exists('$(IceRpcProtobufToolsTaskAssembliesPath)IceRpc.Protobuf.Tools.dll')">
     <OutputFileNamesTask Sources="@(ProtoFile)">
       <Output ItemName="_ProtoFile" TaskParameter="ComputedSources" />
     </OutputFileNamesTask>

